ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -500,8 +500,9 @@ export const GeneDetails = React.memo((
   { gene, compact, showLocusLists, showInlineDetails, individualGeneData, ...labelProps },
 ) => {
   const geneDetails = getDetailSections(GENE_DETAIL_SECTIONS, gene, compact, labelProps, individualGeneData)
-  const geneDiseaseDetails = getDetailSections(GENE_DISEASE_DETAIL_SECTIONS, gene, compact, labelProps)
-  const hasLocusLists = showLocusLists && gene.locusListGuids.length > 0
+  const geneDiseaseDetails = getDetailSections(GENE_DISEASE_DETAIL_SECTIONS, gene, compact, labelProps,
+    null, showInlineDetails)
```

Comment:
this seems like it would break behavior in cases where `showInlineDetails` is already being passed in, can you confirm that in cases where that was already being used nothing is changing?

Reply:
Or I could reuse the `geneModalId` parameter, which is only used by the `Phenotype Prioritized Genes` modal.

Reply:
I think I slightly prefer to add an explicit `noExpand` parameter